docs/getting-started/overview.rst

```diff
@@ -1,6 +1,113 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently 
+supports three main operating systems: Linux, Windows and OS X. 
+
+There are several reasons why we built .NET Core.
```

Comment:
Try to avoid the use of "we". We're wanting to use more inclusive language, with the community. While "we" may be perfectly true now, it probably won't be nearly as accurate 1 year from now. 

Make it less about why we built it and more about the characteristics it has. These documents are not intended to provide a historical account of these products.